app/models/card.rb

```diff
@@ -1,5 +1,9 @@
 class Card < ActiveRecord::Base
   belongs_to :user
+  has_attached_file :card_photo, styles: { medium: "360x360" }
+  validates_attachment_content_type :card_photo,
+                                    content_type: ["image/jpg", "image/jpeg",
+				                   "image/png", "image/gif"]
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Tab detected.